### PR TITLE
document stored procedures and then some

### PR DESCRIPTION
**Pull request by callunity**
Exact same as #1228 , but this time will wait for two approvers.

* also moved the two remaining relevant userspace_properties files (`security.properties` and `submission.properties`) into `miso-web/src/main/resources/` (as per #1212 )
* updated documentation to reflect current tutorial, deployment, and integration testing information

### CODE REVIEW

**Comment by sqwci** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [MisoWebUtils.java#L94](https://github.com/TGAC/miso-lims/blob/1da34ddb562e243cbb8dd5107ff189ce9dbb64f5/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/MisoWebUtils.java#L94): Use the built-in formatting to construct this argument. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: docs/_posts/2016-01-11-admin-manual.md [hunk 1]
```diff
@@ -39,14 +39,16 @@ Development Machine(s):
 * Eclipse
 * A merge tool such as Meld
 
-# Creating a Fork
-Use the GitHub interface or a private instance to create a forked repository.
+<a id="latest-release">
+## Downloading the latest release
+Use the GitHub interface download the [latest release](https://github.com/TGAC/miso-lims/releases/latest).
```
Comment (djcooke) [inline docs/_posts/2016-01-11-admin-manual.md]: to download

File: docs/_posts/2016-01-11-admin-manual.md [hunk 2]
```diff
@@ -39,14 +39,16 @@ Development Machine(s):
 * Eclipse
 * A merge tool such as Meld
 
-# Creating a Fork
-Use the GitHub interface or a private instance to create a forked repository.
+<a id="latest-release">
+## Downloading the latest release
```
Comment (djcooke) [inline docs/_posts/2016-01-11-admin-manual.md]: This line is showing up as a part of the paragraph below it. Same with "Setting Up the Application Server," and "Naming Schemes." I think the link on the line above them is causing the issue. A completely blank line is probably needed above a heading

File: docs/_posts/2016-01-11-admin-manual.md [hunk 3]
```diff
@@ -362,15 +365,16 @@ If you intend to scan Illumina runs, you will have to build `runscanner-illumina
 
 Start Tomcat on this machine.
 
-Edit `miso.properties` and set `miso.runscanner.urls` to the URL of the Run Scanner instance and restart MISO.
+Edit `$CATALINA_HOME/conf/Catalina/localhost/miso.properties` and set `miso.runscanner.urls` to the URL of the Run Scanner instance and restart MISO.
 
 It is possible to set up multiple run scanners managing different sequencers and add all the URLs to `miso.properties`.
 
-You can view the run scanner's state from the main page.
+You can view the run scanner's state from the main page of the runscanner server (http://runscanner.url:8080).
```
Comment (djcooke) [inline docs/_posts/2016-01-11-admin-manual.md]: Probably shouldn't specify the port. It's a product of Tomcat and other server/network config and MISO/runscanner don't affect or care what port is used

File: docs/_posts/2017-12-07-value-type-data.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+---
+
+# Adding Value-Type Data
+
+MISO has several categories of value-type data which the users interact with but cannot add or edit.
+There are some stored procedures which may help in adding these values to the database:
+
+## Indices (also known as Barcodes, Molecular IDs, Tag Barcodes)
+
+An `Index` has a name, a sequence, a position (1 or 2) and is a member of an `Index Family`. And `Index Family` has a name and is associated with a sequencing platform, and can be archived to hide its members when creating new libraries.
```
Comment (djcooke) [inline docs/_posts/2017-12-07-value-type-data.md]: An* Index Family

File: docs/_posts/2017-12-07-value-type-data.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+```
+addSequencerReference(name, platform, instrumentModel, serialNumber, ipAddress, commissionedDate, decommissionedDate, upgradedInstrumentName);
+```
+
+For instance, to add an Illumina HiSeq 2000 which was later upgraded to a HiSeq2500:
+```
+CALL addSequencerReference('h501', 'ILLUMINA', 'Illumina HiSeq 2000', '12345', 'localhost', '2015-01-07', '2016-05-19', 'SN501');
+```
+Note that the sequencing platform must be in all caps, and the instrumentModel must be an exact match for a value in `Platform.instrumentModel`.
+Note also that if adding an instrument which references an upgraded instrument, the upgraded instrument must already exist in MISO.
```
Comment (djcooke) [inline docs/_posts/2017-12-07-value-type-data.md]: Should probably note specifically that `upgradedInstrumentName` can be `NULL`

File: docs/_posts/2017-12-07-value-type-data.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+
+## Sequencing Parameters
+
+Runs have sequencing parameters, and pool Orders are requested with a given set of sequencing parameters. MISO determines whether a pool Order has been completed by looking at the run(s) associated with the pool and seeing if their sequencing parameters match the requested sequencing parameters. While it is possible to add all possible sequencing parameter combinations to this table, it may be easier to add the parameters used most often and describing all infrequently-used parameters with the 'Custom' label.
+
+```
+addSequencingParamters(name, platform, instrumentModel, readLength, paired, chemistryVersion);
```
Comment (djcooke) [inline docs/_posts/2017-12-07-value-type-data.md]: Parameters

File: sqlstore/src/main/resources/db/migration_beforeMigrate/05_add_platform.sql [hunk 7]
```diff
@@ -1,8 +1,10 @@
+-- If you change these method names or signatures, update the docs in `docs/_posts/2017-12-07-value-type-data.md`
 -- StartNoTest
 DELIMITER //
 
 DROP PROCEDURE IF EXISTS addPlatform//
-CREATE PROCEDURE addPlatform(
+DROP PROCEDURE IF EXISTS addInstrumentModel//
+CREATE PROCEDURE addInstrumentModel(
```
Comment (djcooke) [inline sqlstore/src/main/resources/db/migration_beforeMigrate/05_add_platform.sql]: Rename breaks OICR after-migrate script, so should have a PR there as well: https://github.com/oicr-gsi/miso-lims/blob/oicr/sqlstore/src/main/resources/db/migration_afterMigrate/65_sequencers.sql

Comment (callunity) [reply sqlstore/src/main/resources/db/migration_beforeMigrate/05_add_platform.sql]: OICR PR: https://github.com/oicr-gsi/miso-lims/pull/210

**Pull request merged**